test/image/README.md

```diff
@@ -1,24 +1,46 @@
 # plotly.js image testing
 
-Test plotly.js with the Plotly Image-Server docker container.
+Test plotly.js with Plotly's image testing docker container.
 
+Requirements:
+- `docker` | [installation guidelines](http://docs.docker.com/engine/installation/)
+- `docker-machine` (for Mac and Windows users only) | [installation guidelines](https://docs.docker.com/machine/install-machine/)
+- `docker-compose` | [installation guidelines](https://docs.docker.com/compose/install/)
+
+### Step 0: Start the docker machine (Mac and Windows users only)
+
+Boot up docker machine (named `default`):
+
+```bash
+docker-machine start default
+```
+
+Set up the docker environment for `docker-compose`:
+
+```bash
+eval $(docker-machine env default)
```

Comment:
yup, the eval did it for me. (I used it with quotes, as mentioned within the output of `docker-machine env default` itself)
Image tests work locally for me now! Well almost, some intermittent gl3d errors (`gl3d_marker-color.png` failed twice in a row, `gl3d_mirror-ticks.png` failed one of two times)

Does it save the images it makes somewhere when they fail?

Version: 1.5.1, just like @cldougl (installed today)